components/card/card.js

```diff
@@ -1 +1,25 @@
-export function createCharacterCard() {}
+export function createCharacterCard(name, status, type, occurrences, src) {
+  const cardContainer = document.querySelector('["card-container"]');
```

Comment:
You do not need the cardContainer here 🙂 And it is already queried in the `index.js` as well.

```suggestion

```